nautobot/extras/api/views.py

```diff
@@ -381,3 +387,27 @@ class RelationshipAssociationViewSet(ModelViewSet):
     queryset = RelationshipAssociation.objects.all()
     serializer_class = serializers.RelationshipAssociationSerializer
     filterset_class = filters.RelationshipAssociationFilterSet
+
+
+#
+# GraphQL Queries
+#
+
+
+class GraphQLQueryViewSet(ModelViewSet):
+    queryset = GraphQLQuery.objects.all()
+    serializer_class = serializers.GraphQLQuerySerializer
+    filterset_class = filters.GraphQLQueryFilterSet
+
+    @swagger_auto_schema(method="post", request_body=serializers.GraphQLQuerySerializer)
+    @action(detail=True, methods=["post"])
+    def run(self, request, pk):
+        query_object = get_object_or_404(self.queryset, slug=pk)
```

Comment:
I think it would be great to create some util functions to execute a GraphQL query and another one to execute saved GraphQL query internally from the ORM. 
It will be very useful to reuse saved queries from a plugin.